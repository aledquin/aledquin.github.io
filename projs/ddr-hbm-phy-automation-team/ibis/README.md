# README

IBIS Standards: [IBIS.ORG](https://ibis.org/)

DDR Design Automation Workgroup - [Confluence](https://jiradocs.internal.synopsys.com/display/SG/Coding+Best+Practices+and+Recommendations)

Coding Standards - [Confluence](https://jiradocs.internal.synopsys.com/display/SG/Coding+Best+Practices+and+Recommendations)

Script Usage Documentation: [QA Scripts Documentation](https://synopsys-my.sharepoint.com/:p:/p/wadhawan/EQXoE4dKwCVFnDL-4rcqo0YBPBMvce7VVARAe6ZUsWrq-Q?e=oB7S2F)

# What is IBIS-QA? 

The acronym IBIS stands for I/O Buffer Information Specification. IBIS is a standard for electronic behavioral specifications of integrated circuit input/output analog characteristics. IBIS modelfiles are generated by the IBIS development team in Armenia. This repository hosts a variety of tools for the purposes of quality assurance of these IBIS modelfiles and related IBIS use cases such as extracting calibration codes from bbSim generated data.

# How to use tools inside this repository?

All tools are realesed to the Shelltools area. The latest IBIS release can be obtained by invoking `module load ibis` in the Synopsys environment.

# How to contribute to this repository?

Clone this repository using the SSH URL. For more information on how to clone and obtian access to GitLab see Workgroup Confluence pages linked above. It is recommended to setup SSH keys for contributions. [SSH SETUP](https://docs.gitlab.com/ee/ssh/)

# Repository Do(s) and Don't(s)

1. Do contribute to this repository to keep the tools up-to-date.
2. Do become a steward of the codebase and help keep it clean and organised.
3. Do write tests and take advantage of shared library routines. 





