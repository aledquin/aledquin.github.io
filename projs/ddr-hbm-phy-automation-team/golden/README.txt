The purpose of this golden directory is to store test results that are considered to pass.
Next time we run the same set of tests we have something to compare it against.
We can then perform a diff operation on the current results vs the golden results.
This will tell us if something has changed since the last time we updated the
golden results.


